pkg/cli/admin/createbootstrapprojecttemplate/sample_template.go

```diff
@@ -5,11 +5,11 @@ import (
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/runtime/serializer"
 	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
-	rbacv1helpers "k8s.io/kubernetes/pkg/apis/rbac/v1"
```

Comment:
Drop this entire commit, it's not needed, not yet. Upstream is suffering a similar problem here. We'll move when it'll be solved upstream. I don't want to copy that code.

Reply:
ok sounds good, thanks